packages/launch/src/focused-launch/summary/index.tsx

```diff
@@ -352,44 +360,79 @@ const PlanStep: React.FunctionComponent< PlanStepProps > = ( {
 						<div>
 							<FocusedLaunchSummaryItem
 								isLoading={ ! defaultFreePlan || ! defaultPaidPlan }
-								readOnly={ hasPaidDomain || selectedPaidDomain }
-								onClick={ () => defaultFreePlan && setPlan( defaultFreePlan ) }
-								isSelected={ ! ( hasPaidDomain || selectedPaidDomain ) && selectedPlan?.isFree }
+								onClick={ () => defaultPaidPlan && setPlan( defaultPaidPlan ) }
+								isSelected={ selectedPlan?.storeSlug === defaultPaidPlan?.storeSlug }
 							>
 								<LeadingContentSide
 									label={
 										/* translators: %s is WordPress.com plan name (eg: Premium Plan) */
-										sprintf( __( '%s Plan', __i18n_text_domain__ ), defaultFreePlan?.title ?? '' )
+										sprintf( __( '%s Plan', __i18n_text_domain__ ), defaultPaidPlan?.title ?? '' )
+									}
+									badgeText={
+										defaultPaidPlan?.isPopular ? __( 'Popular', __i18n_text_domain__ ) : ''
 									}
 								/>
-								<TrailingContentSide
-									nodeType={ hasPaidDomain || selectedPaidDomain ? 'WARNING' : 'PRICE' }
-								>
-									{ hasPaidDomain || selectedPaidDomain
-										? __( 'Not available with your domain selection', __i18n_text_domain__ )
-										: __( 'Free', __i18n_text_domain__ ) }
+								<TrailingContentSide nodeType="PRICE">
+									<span>{ defaultPaidPlan && planPrices[ defaultPaidPlan?.storeSlug ] }</span>
+									<span>
+										{
+											// translators: /mo is short for "per-month"
+											__( '/mo', __i18n_text_domain__ )
+										}
+									</span>
 								</TrailingContentSide>
 							</FocusedLaunchSummaryItem>
+
+							{ nonDefaultPaidPlan && (
+								<FocusedLaunchSummaryItem
+									isLoading={ ! defaultFreePlan || ! defaultPaidPlan }
+									onClick={ () => nonDefaultPaidPlan && setPlan( nonDefaultPaidPlan ) }
+									isSelected={ selectedPlan?.storeSlug === nonDefaultPaidPlan?.storeSlug }
```

Comment:
Similarly to previous comment, the `isSelected` check could potentially return `true` if both `selectedPlan` and `nonDefaultPaidPlan ` are `undefined`. We should probably change this check to something like
```suggestion
									isSelected={ selectedPlan && selectedPlan.storeSlug === nonDefaultPaidPlan?.storeSlug }
```